index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>JS Documentation</title>
+	<link rel="stylesheet" href="style.css">
+</head>
+<body>
+	<main id="main-doc">
+		<!-- navbar -->
+			<nav id="nav">
+				<h2 style="padding-left: 2rem;">JS Documentation</h2>
+				<hr>
+				<ul>
+					<li class="nav-items"><a href="#introduction">Introduction</a></li>
+					<li class="nav-items"><a href="#alreadyKnow">What you should already know</a></li>
+					<li class="nav-items"><a href="#java">Javascript and Java</a></li>
+					<li class="nav-items"><a href="#helloworld">Hello World</a></li>
+					<li class="nav-items"><a href="#variables">Variables</a></li>
+					<li class="nav-items"><a href="#declare">Declaring variables</a></li>
+					<li class="nav-items"><a href="#score">Variables Score</a></li>
+				</ul>
+			</nav>
+<!-- main section -->
+			<div id="mainSection">
+				<div id="introduction">
+					<h1>Introduction</h1>
+					<p>JavaScript is a cross-platform, object-oriented scripting language. It is a small and lightweight language. Inside a host environment (for example, a web browser),
+						JavaScript can be connected to the objects of its environment to provide programmatic control over them.
+					</p>
+					<p>
+						JavaScript contains a standard library of objects, such as Array, Date, and Math, and a core set of language elements such as operators, control structures, and statements.
+						Core JavaScript can be extended for a variety of purposes by supplementing it with additional objects; for example:
+					</p>
+					<ul>
+						<li>
+							Client-side JavaScript extends the core language by supplying objects to control a browser and its Document Object Model (DOM). For example, client-side
+							extensions allow an application to place elements on an HTML form and respond to user events such as mouse clicks, form input, and page navigation.
+						</li>
+						<li>
+							Server-side JavaScript extends the core language by supplying objects relevant to running JavaScript on a server. For example,
+							server-side extensions allow an application to communicate with a database, provide continuity of information from one invocation to another of the
+							application, or perform file manipulations on a server.
+						</li>
+					</ul>
+				</div>
+
+				<div id="alreadyKnow">
```

Comment:
Please write relevant naming for the class and id

Reply:
As mentioned in my previous comment, I kindly suggest 🙏 including an explanation or an example that illustrates the usefulness of the suggested action. This approach ensures a more collaborative and informative exchange of feedback. 👍💡 It greatly enhances the overall effectiveness of the feedback process. 👍